src/pages/_app.js

```diff
@@ -79,13 +80,15 @@ Layout.propTypes = {
 const queryCache = new QueryCache();
 const queryClient = new QueryClient({ queryCache });
 
+const isBrowser = () => typeof window !== 'undefined';
```

Comment:
We already have an useIsClient hook.

Reply:
As discussed over Slack I'm not going to use that for now because it results in errors again. (The first 1-2 renders of `App` on the client-side get `false` from `useIsClient` for some reason.)
We'll look into that later.